server/controllers/api_controller.go

```diff
@@ -208,6 +210,13 @@ func (a *APIController) apiPlan(request *APIRequest, ctx *command.Context) (*com
 	return &command.Result{ProjectResults: projectResults}, nil
 }
 
+func GetOrElse(val, def string) string {
```

Comment:
I don't think this function needs to be exported; we probably don't want other packages using this/depending on its behavior not changing.
```suggestion
func getOrElse(val, def string) string {
```